docs/modules/ROOT/pages/cpp-client-getting-started.adoc

```diff
@@ -0,0 +1,203 @@
+= Getting Started with the Hazelcast C++ Client
+:page-layout: tutorial
+:page-product: platform
+:page-categories: Caching, Getting Started
+:page-lang: cplus
+:page-est-time: 5-10 mins
+:description: This tutorial will get you started with the Hazelcast C++ client.
+
+== What You'll Learn
+
+{description}
+
+== Before you Begin
+
+* A text editor or IDE
+* Docker
+* C++ 11+
+* Vcpkg
+
+== Start a Hazelcast Member
+
+We will use the 5.1 version of Hazelcast for this tutorial.
+
+In this tutorial, we will use Docker for simplicity. You can also use https://docs.hazelcast.com/hazelcast/5.1/getting-started/get-started-cli[CLI], https://docs.hazelcast.com/hazelcast/5.1/getting-started/get-started-binary[Binary] and https://docs.hazelcast.com/hazelcast/5.1/getting-started/get-started-java[Maven].
+
+[source,bash]
+----
+docker run -p 5701:5701 hazelcast/hazelcast:5.1
+----
+
+This will start a new Hazelcast member at port 5701. Now, we have a Hazelcast cluster with just one member.
+
+== Install Hazelcast C++ Client
+In this tutorial we will use Vcpkg for installing the C++ client. You can also use https://github.com/hazelcast/hazelcast-cpp-client/blob/master/Reference_Manual.md#111-conan-users[Conan] or install from source using CMake as explained https://github.com/hazelcast/hazelcast-cpp-client/blob/master/Reference_Manual.md#113-install-from-source-code-using-cmake[here].
+
+Before starting download and install Vcpkg itself if you haven't already:
+[source,bash]
+----
+git clone https://github.com/microsoft/vcpkg
+.\vcpkg\bootstrap-vcpkg.bat
+----
+
+First, execute the following to install `hazelcast-cpp-client` ith its `boost` dependencies:
+
+[source,bash]
+----
+.\vcpkg\vcpkg install hazelcast-cpp-client
+----
+After the installation, the library is available for usage.
+For example, if you are using CMake for your builds, you can use the following cmake build command with the `CMAKE_TOOLCHAIN_FILE` cmake option to be the `vcpkg.cmake`.
+
+[source,bash]
+----
+cmake -B [build directory] -S . -DCMAKE_TOOLCHAIN_FILE=[path to vcpkg]/scripts/buildsystems/vcpkg.cmake
```

Comment:
you did not use the `[path to vcpkg]` in the above but just used `\vcpkg` directory name. Can you make them consistent?